libs/components/layout/src/lib/modules/action-button/action-button.component.scss

```diff
@@ -15,29 +36,9 @@
 
 .sky-action-button-icon-header-container {
   display: flex;
+  flex-direction: var(--sky-action-button-icon-header-container-flex-direction);
   justify-content: center;
-}
-
-@include mixins.sky-host-responsive-container-xs-min(false) {
-  .sky-action-button {
-    padding: $sky-padding-double $sky-padding-double $sky-padding-triple;
```

Comment:
Inside a modal, the styles for both the `xs-min` responsive container (from the modal) and the `sm-min` responsive container (from the body element) were applied, and since no `max-width` or `flex-direction` were specified for the `xs-min` container, it inherited the `sm-min`. The fix was to always specify these rules. I moved them into CSS custom properties since the only thing that differed after this change were the values (the selectors were identical other than the responsive container mixin).

Reply:
No, just small and medium modals. Large modals exceed the extra-small breakpoint size, so action buttons should display side-by-side.

Reply:
This is what I see on the playground application, when opening a "large" modal:
<img width="942" alt="Screenshot 2023-10-10 at 11 02 42 AM" src="https://github.com/blackbaud/skyux/assets/12497062/6e761246-38e9-43bb-943c-3163dd43ed97">

Reply:
Sorry, my response was for default theme which this code change addresses. Action buttons are wider in modern theme and don't fit side-by-side in a large modal. This is the expected behavior.

Reply:
Understood. Thanks for clarifying!